src/app/home/home.component.ts

```diff
@@ -132,7 +134,7 @@ export class HomeComponent implements OnInit {
 	redirect() {
 		this.cookieService.set('org', this.org.url, null, '/', '.newvote.org');
 		// window.location.href = this.org.authUrl;
-		window.open(this.org.authUrl, '_self');
+		this.router.navigate(['/auth/login'])
 	}
```

Comment:
We shouldn't set the cookie here anymore, if we're simply redirecting with the router then move the redirect into a `routerLink` property in the .html template and just remove this redirect method, it's redundant now.